src/SPIN/pair_spin_exchange.cpp

```diff
@@ -507,10 +560,12 @@ void PairSpinExchange::read_restart_settings(FILE *fp)
 {
   if (comm->me == 0) {
     utils::sfread(FLERR,&cut_spin_exchange_global,sizeof(double),1,fp,nullptr,error);
+    utils::sfread(FLERR,&e_offset,sizeof(int),1,fp,nullptr,error);
```

Comment:
This has backward compatibility implications. Older restarts won't be read correctly.

Reply:
Yes, this is correct Richard. I added it in the backward compatibility section of the PR notice.